docs/guides/development/troubleshooting.md

```diff
@@ -53,6 +53,13 @@ These errors can occur when installing the SDK locally, specifically during the
 This error will appear at the bottom of the screen after pressing a button, and indicates that the formula used in the button isn't a valid action. In order for a formula to be used as an action within a button, it must be defined with `isAction: true`. See the [Actions guide][actions_create] for more information.
 
 
+### HTTP response too large
+
+- `RESOURCE_EXHAUSTED: Received message larger than max`
```

Comment:
We do have a task to detect this error and turn it into something comprehensible and actionable.

Reply:
Added a comment linking this to the bug.